### PR TITLE
Fix IntegrationTestingTutorial network map race condition

### PR DESCRIPTION
**Pull request by Clintonio**
Someone should own this particular bug. Will be reporting as an issue.

### CODE REVIEW

**Comment by exFalso** [comment]
We shouldn't really ignore the tutorial test...
Don't we just need to call `waitUntilRegisteredWithNetworkMap()` on `alice`/`bob`?

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt [hunk 1]
```diff
@@ -51,8 +52,11 @@ class IntegrationTestingTutorial {
 
             val bobClient = bob.rpcClientToNode()
             val bobProxy = bobClient.start("bobUser", "testPassword2").proxy
+
+            Futures.allAsList(aliceProxy.waitUntilRegisteredWithNetworkMap(), bobProxy.waitUntilRegisteredWithNetworkMap()).get()
```
Comment (exFalso) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: Small thing: it should be `getOrThrow()`

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt [hunk 2]
```diff
@@ -22,6 +22,7 @@ import net.corda.testing.expect
 import net.corda.testing.expectEvents
 import net.corda.testing.parallel
 import net.corda.testing.sequence
+import org.junit.Ignore
```
Comment (exFalso) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: Not needed anymore

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt [hunk 3]
```diff
@@ -51,8 +52,11 @@ class IntegrationTestingTutorial {
 
             val bobClient = bob.rpcClientToNode()
             val bobProxy = bobClient.start("bobUser", "testPassword2").proxy
+
+            Futures.allAsList(aliceProxy.waitUntilRegisteredWithNetworkMap(), bobProxy.waitUntilRegisteredWithNetworkMap()).get()
             // END 2
 
+
```
Comment (exFalso) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: whitespace 🚓

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt (hunk 1 above)
Comment (Clintonio) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: Fixed.

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt (hunk 2 above)
Comment (Clintonio) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: Good spot.

File: docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt (hunk 3 above)
Comment (Clintonio) [inline docs/source/example-code/src/integration-test/kotlin/net/corda/docs/IntegrationTestingTutorial.kt]: I got policed! (Fixed)

**Comment by Clintonio** [comment]
Back to you @exFalso

**Pull request merged**